src/main/java/fr/insee/queen/api/surveyunittempzone/repository/SurveyUnitTempZoneDao.java

```diff
@@ -1,16 +1,18 @@
 package fr.insee.queen.api.surveyunittempzone.repository;
 
+import fr.insee.queen.api.surveyunittempzone.repository.entity.SurveyUnitTempZoneDB;
 import fr.insee.queen.api.surveyunittempzone.repository.jpa.SurveyUnitTempZoneJpaRepository;
 import fr.insee.queen.api.surveyunittempzone.service.gateway.SurveyUnitTempZoneRepository;
 import fr.insee.queen.api.surveyunittempzone.service.model.SurveyUnitTempZone;
 import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
-import java.util.UUID;
 
 /**
- * SurveyUnitTempZone is the repository using to save surveyUnit with probleme in DB
+ * Repository to handle survey units in temp zone.
+ * A survey unit going the temporary zone is a survey unit which cannot be found as a survey unit during an interviewer synchronisation
```

Comment:
the SU can still exist , it happends when the SU is not affected to the interviewer when he/she put SU